bulk_lookup/views.py

```diff
@@ -107,45 +99,60 @@ def get_context_data(self, form, **kwargs):
             raise WizardError
         context['num_good_rows'] = pc_data['num_rows'] - pc_data['bad_rows']
         if self.steps.current == 'personal_details':
-            context['price'] = settings.BULK_LOOKUP_PRICE
+            context['price'] = settings.BULK_LOOKUP_AMOUNT
             if self.object and self.object.plan.id == settings.PRICING[-1]['plan']:
```

Comment:
As the looking up of self.object above in get_form_kwargs was removed, this was never true - added a similar is_authenticated/lookup of self.object here afterwards.